### ISSUE TITLE
add ability to reference same footnote multiple times

### ISSUE THREAD

**Issue by nyancrimew** [KittyGiraudel/eleventy-plugin-footnotes#83]
it would be nice if instead of simply not rendering at all, footnoterefs without a content but sharing an anchor with an existing footnote instead linked to the same footnote. a usecase of this would for example be a news article mentioning multiple different sources for whom pseudonyms are used, a single footnote could be used to explain this for all of them.

**Comment by nyancrimew** [comment]
that is me indeed (and im [back on twitter again](https://twitter.com/awawawhoami) btw), would be cool to see if this works, though i've now realized it would complicate the whole css counter for numbered links thing but i'll see about that once it actually becomes a problem :p 

and no hurry!

**Comment by KittyGiraudel** [comment]
Alright, I tried to think a bit about this. There are 2 problems to make it work — one simple to solve, and one not:

1. Right now, a footnote reference without a content argument is skipped entirely. We could easily have an option to challenge that and let “dead-end” footnote refs still be rendered. Easy.
2. The whole premise is to leverage CSS counters to for the numbering, so we don’t cary a bunch of logic to reproduce that behavior. As per the suggestion, we increment the counter with every footnote reference. So even if refs A, B, C all point to the same footnote and only A defines the footnote content, B and C being void, we still have A bumping the counter by 1, B by 1 and C by 1. We don’t have a concept of unicity on attributes.

I don’t know if it’s solvable without dropping CSS counters entirely.

**Comment by nyancrimew** [comment]
yeah 2. is also what i have been worried about, the only thing i could think of for it is potentially some really cursed set-up with counter-set? but i havent really looked into how possible it would be
